File: extentmerge/src/main/java/com/relevantcodes/extentreports/source/Table.java
package com.relevantcodes.extentreports.source;

public class Table {
	public static String getRowSource(int cols) {
		String td = "";
		
		for (int ix = 0; ix < cols; ix++) {
			td += "<td></td>";
		}
		
		return "<table><tr>" + td + " </tr></table>";
	}
}

File: dotnet/RelevantCodes.ExtentReports/RelevantCodes.ExtentReports/View/Table.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RelevantCodes.ExtentReports.View
{
    internal class Table
    {
        public static string Source
        {
            get
            {
                return @"
                        <table></table>
                        ";
            }
        }

        public static string RowSource
        {
            get
            {
                return @"
                        <tr></tr>
                        ";
            }
        }

        public static string CellSource(int Cols)
        {
            string td = "";

            for (int ix = 1; ix <= Cols; ix++)
            {
                td += "<td></td>";
            }

            return td;
        }
    }
}
